WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentPreflightController.swift

```diff
@@ -35,10 +35,19 @@ final class CardPresentPaymentPreflightController {
     ///
     private let analytics: Analytics
 
+    /// Root View Controller
```

Comment:
A super not a blocker 😅  Do we need to repeat the property name in the comment? e.g `/// Root View Controller`

Reply:
Not really. It's just for consistency with the rest, which is inherited from the code that's been copied and refactored from the existing flows. It's weird if we have the comments for some of the properties but not others. I don't think we have a very consistent style through the app for this though.